Cargo.toml

```diff
@@ -1,17 +1,7 @@
-[package]
-name = "oba-services"
-version = "0.1.0"
-authors = ["josojo <josojo@hotmail.de>"]
-edition = "2018"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
-[dependencies]
-ethcontract = "0.8.0"
-serde = { version = "1.0", features = ["derive"] }
-rustc-hex = "2.1.0"
-anyhow = "1.0.32"
-serde_json = "1.0.57"
-parking_lot = "0.11.0"
-warp = "0.2"
-tokio = { version = "0.2.4", features = ["macros", "test-util"] }
+[workspace]
+members = [
+    "driver",
+]
+default-members = [
+    "driver",
+]
```

Comment:
`default-members` is not needed here, by default all `members` are included :smile: